src/components/Components/CandidateSection/CandidateSection.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import "./CandidateSection.scss"
+import ViewMoreIcon from "./../../icons/Vector.png"
```

Comment:
change icon location to a folder assets inside the component. but this is global icon what to then @jasper-prgh

Reply:
then create a new assets folder in src/components/assets

Reply:
also change the filename to soemthing more identifying and use camelCase please